src/shared/layouts/MainLayout.tsx

```diff
@@ -55,6 +58,8 @@ const MainLayout: React.FunctionComponent<MainLayoutProps> = ({
     })
   );
 
+  console.log('gtmCode', gtmCode);
```

Comment:
Do we need to log the code in to console?

Reply:
It is a draft PR, will be removed